### PR TITLE
Remove useless minver configuration in functional tests

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Description

This remove the unneeded min_ver configuration in functional tests and rename tests when required. We're only supporting python version > 3.6 so this became useless configuration.

### CODE REVIEW

File: tests/functional/s/singledispatch_functions_py3.rc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[testoptions]
-;https://bugs.python.org/issue10445
```
Comment (cdce8p) [inline tests/functional/s/singledispatch_functions_py3.rc:2]: Can you add this as a comment in the test file?

File: tests/functional/a/abstract/abstract_class_instantiated.py [hunk 2]
```diff
@@ -126,3 +126,18 @@ def test(self):
...
+    defines abstract methods.
+    """
+    @abc.abstractmethod
+    def test(self):
+        pass
+
+
+def main_two():
+    """ do nothing """
+    BadClass() # [abstract-class-instantiated]
```
Comment (cdce8p) [inline tests/functional/a/abstract/abstract_class_instantiated.py:143]:
```suggestion
    BadClassTwo()  # [abstract-class-instantiated]
```
?

**Review by cdce8p — APPROVED** [review]
Great cleanup!

File: tests/functional/s/singledispatch_functions_py3.rc (hunk 1 above)
Comment (Pierre-Sassoulas) [reply tests/functional/s/singledispatch_functions_py3.rc:2]: The content of the link is "python 3.2 and 3.3 receive only security update, we can't backport the fix that is in 3.4 in it", so we can remove the link too.

File: tests/functional/a/abstract/abstract_class_instantiated.py (hunk 2 above)
Comment (Pierre-Sassoulas) [reply tests/functional/a/abstract/abstract_class_instantiated.py]: Nice catch, 👍

**Pull request merged**